api/proto/teleport/devicetrust/v1/device.proto

```diff
@@ -90,22 +90,18 @@ message Device {
 
 // DeviceCredential represents the current enrolled public key of a device.
 message DeviceCredential {
+  reserved 5;
```

Comment:
Because we never really used the field I think it's OK to just drop it.

If you want to keep the reserved, I suggest adding:

```suggestion
  reserved 5; // bytes tpm_attestation_key_der
```

(not a big deal, though)

Reply:
Sure - more than happy to just drop it I was just being overly cautious.

Reply:
3a7a39d8b3